tests/agents/test_gpt4_agent.py

```diff
@@ -80,14 +100,46 @@ def test_tick(log, complete_text, mock_gpt4_agent, gpt4agent, client):
     # Assert the response status code and content
     assert response.status_code == 200
     response_payload = response.json()
+
     print(response_payload)
     assert response_payload == {
-    'world_context': [
-        "You are a meticulous thinker. \nGiven what you know about the world today, and the main task that you need to complete, consider if there are any additional facts that you should add to the list of things you consider. \nDo not add anything that doesn't need to be added, consolidate anything that is worth consolidating with simpler statements.WORLD_CONTEXT:TASK_CONTEXT:Write a haiku:completion1",
-        "You are a meticulous thinker. \nGiven what you know about the world today, and the main task that you need to complete, consider if there are any additional facts that you should add to the list of things you consider. \nDo not add anything that doesn't need to be added, consolidate anything that is worth consolidating with simpler statements.WORLD_CONTEXT:TASK_CONTEXT:Write a haiku:completion2"
-    ],
+    'world_context': [],
     'task_context': [],
     'execution_context': []
     }
-    assert log.call_count == 1
+    assert log.call_count == 2
+
+@patch('app.main.get_gpt4_client')
+@patch('agents.gpt4_agent.GPT4Agent')
+@patch('agents.gpt4_agent.GPT4Agent.complete_text')
+@patch('adapters.log_adapter.LogAdapter.log')
+def test_tick_with_prompt_and_world_processing_parameters(log, complete_text, mock_gpt4_agent, mock_gpt4_agent2, gpt4agent, client, process_world_variation_gpt4agents):
```

Comment:
TODO: Need to add some additional wire through with testing the world processing in this multi-test. Context is being pulled from the DB currently.